Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1683.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using System.Collections.Generic; 
+
+#if UITEST
```

Comment:
If you opt to not create a UI test, then we should just remove these unnecessary `if` defs. :)